doc/articles/get-started-dotnet-new.md

```diff
@@ -22,8 +22,6 @@ This template can be used to create a blank multi-platform application for iOS,
 
 This template uses a single project head for iOS, Android, macOS, and Mac Catalyst. It requires Visual Studio 2022.
 
-[**Find detailed instructions here.**](get-started-winui3.md)
```

Comment:
Line 59 has the same link, as well as `updating-to-winui3.md` and `uwp-vs-winui3.md`